packages/calculator/src/dragablecalculator.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Calculator from '@pie-framework/material-ui-calculator';
+import { withStyles } from 'material-ui/styles';
+import Draggable from 'react-draggable';
+
+const styles = {
+    handle: {
+        border: "1px solid black",
+        zIndex: "10",
+        position: "absolute",
+        cursor: "move"
+    }
+  };
+  
+
+class DraggableCalculator extends React.Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        const { mode } = this.props;
+        const { show } = this.props;
+        return (
+            <div>
+                {show &&
+                    (<Draggable
+                        defaultPosition={{ x: 0, y: 0 }}>
```

Comment:
would it make sense to store these in state? so that if he user wants to show the calculator again it appears in the same position?